DigitalLearningSolutions.Web/ViewModels/LearningContent/LearningContentViewModel.cs

```diff
@@ -3,12 +3,14 @@
     public class LearningContentViewModel
     {
         public readonly string Brand;
+        public readonly bool LongTitle;

```

Comment:
It's nice to have boolean variable names as statements, so it would be better to call this (and the constructor param) something like `titleIsLong`.

That's so that it's obvious 
- that they're booleans (`longTitle` sounds a bit like it refers to an extended version of the title)
- what it means if they're true or false (in this case it's pretty obvious)

Reply:
Makes perfect sense, changed the name to titleIsLong